controllers/controllers.js

```diff
@@ -14,6 +14,17 @@ exports.getArticleById = (req,res, next)=>{
     .catch(next)
 }
 
+exports.getAllArticles = (req,res,next)=>{
+    fetchAllArticles().then(articles => {
+        articles.map(article=>{
+            article.comment_count = Number(article.comment_count)
```

Comment:
Nice! you could also use the SQL CAST() function in your model to do this. Either works 👍